MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include kernel_gateway/jupyter_websocket/swagger.*
```

Comment:
This is probably a leftover copy/paste.

Reply:
Yep.  I had reviewed the `MANIFEST.in` from Enterprise Gateway, but used the one from Kernel Gateway - nice catch - thanks!